jetpack-beta.php

```diff
@@ -857,6 +887,120 @@ static function should_update_dev_to_master() {
 		$manifest = self::get_beta_manifest();
 		return ! isset( $manifest->{$section}->{$branch} );
 	}
+
+	static function is_set_to_autoupdate() {
+		return get_option( 'jp_beta_autoupdate', false );
```

Comment:
I find it a little bit weird that `jp_beta_autoupdate` is used for both updating the plugin, and updating the Jetpack plugin version one is using. but it has just worked